components/clinical/shared/type-summary/package.json

```diff
@@ -34,7 +34,8 @@
     "@ltht-react/styles": "^2.0.55",
     "@ltht-react/types": "^2.0.55",
     "@ltht-react/utils": "^2.0.55",
-    "react": "^18.2.0"
+    "react": "^18.2.0",
+    "date-fns": "^2.22.1"
```

Comment:
Reason for this change?

Reply:
throwing missing package error on lint

Reply:
I think this might be related to some work @airemechak has done - just tagging for visibility in case they were already looking at it

Reply:
Late arrival but yes that date-fns addition is me, as part of this PR https://github.com/ltht-epr/ltht-react/pull/352